shopfloor_reception/static/description/index.html

```diff
@@ -367,7 +367,7 @@ <h1 class="title">Shopfloor Reception</h1>
 !! This file is generated by oca-gen-addon-readme !!
 !! changes will be overwritten.                   !!
 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-!! source digest: sha256:339c718c4454459205067e05b3ff5c6dac381126cd1aaaf9f7503be87b410a62
```

Comment:
just rollback this change and always use this env key:

```export SKIP=oca-gen-addon-readme``` 

Source https://github.com/OCA/wms/blob/16.0/.github/workflows/pre-commit.yml#L38